### PR TITLE
Misha/ob-159_reverse_a_string

### PR DESCRIPTION
**Pull request by krassovki**

### CODE REVIEW

File: Misha/OB-159_reverse_a_string/string_reverse.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+def string_reverser(string):
```
Comment (glock3) [inline Misha/OB-159_reverse_a_string/string_reverse.py:1]: Please move your task to String folder

File: Misha/OB-159_reverse_a_string/string_reverse_test.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import unittest
+from string_reverse import string_reverser
+
+
+class StringReverserTest(unittest.TestCase):
+
+    def test_string_reverser_with_none_value(self):
```
Comment (glock3) [inline Misha/OB-159_reverse_a_string/string_reverse_test.py:7]: Please add test with mirrored special character
`\\n`
`\\0\\t\\n`

File: Misha/OB-159_reverse_a_string/string_reverse.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    if string is None:
+        raise TypeError('Non-type value')
+    else:
+        reversed_string = ''
+        string_size = len(string)
+        print('string to reverse: [' + string + ']')
+        for index in range(string_size):
+            reversed_string += string[string_size - index - 1]
+        print('reversed string: [' + reversed_string + ']')
+        return reversed_string
```
Comment (glock3) [inline Misha/OB-159_reverse_a_string/string_reverse.py:11]: Good job! Solution is testes and looks like work correctly! But could you do it in one line please? :)

**Review by glock3 — CHANGES_REQUESTED** [review]

File: Misha/OB-159_reverse_a_string/string_reverse.py (hunk 3 above)
Comment (krassovki) [reply Misha/OB-159_reverse_a_string/string_reverse.py:11]: i thought i should do it manually, not with in-built functions

**Pull request closed without merging**